application/deed/utils.py

```diff
@@ -101,4 +108,81 @@ def load_json_file(file_path):
     return json_data
 
 
+def convert_json_to_sig_slot(borrower_json):
```

Comment:
I have no idea what a sig slot is, either better descriptive name or a comment please